zb-node.js

```diff
@@ -96,6 +97,22 @@ class ZigbeeNode extends Device {
     this.classified = false;
   }
 
+  updateAddr16(addr16) {
+    if (addr16 &&
+        this.addr16 != addr16 &&
+        addr16.match(/^[0-9a-f]{4}$/) &&
+        addr16 < 'fffc') {
```

Comment:
Can we not rely on implicit type coercion and instead do `parseInt(addr16, 16) < 0xfffc`?